### PR TITLE
IGVF-1831-st-outputs

### PR DESCRIPTION
**Pull request by ian-whaling**

### CODE REVIEW

File: src/igvfd/schemas/configuration_file.json [hunk 1]
```diff
@@ -42,7 +42,26 @@
...
+                }
+            },
+            "then": {
+                "not": {
+                    "required": [
+                        "seqspec_of"
+                    ]
+                }
+            }
+        }
```
Comment (shengchd) [inline src/igvfd/schemas/configuration_file.json:64]: would it be reasonable to add some dependencies between `content_type` and `file_format`?

File: src/igvfd/tests/data/inserts/matrix_file.json [hunk 2]
```diff
@@ -42,5 +42,27 @@
...
+        "lab": "christina-leslie",
+        "award": "1U01HG012103-01",
+        "aliases": [
+            "igvf:raw_feature_barcode_matrix"
+        ],
+        "status": "released",
+        "release_timestamp": "2024-07-06T12:34:56Z",
+        "md5sum": "e2cd3246b8a725eba73deaf9755785cd",
+        "file_format": "h5ad",
+        "content_type": "sparse peak count matrix",
```
Comment (shengchd) [inline src/igvfd/tests/data/inserts/matrix_file.json:57]: did you want to change the `content_type` example here?

**Review by shengchd — CHANGES_REQUESTED** [review]
commented on dependencies under configuration file, and a fix for test data.

File: src/igvfd/tests/data/inserts/matrix_file.json (hunk 2 above)
Comment (ian-whaling) [reply src/igvfd/tests/data/inserts/matrix_file.json:57]: Yes, changed.

File: src/igvfd/schemas/configuration_file.json (hunk 1 above)
Comment (ian-whaling) [reply src/igvfd/schemas/configuration_file.json:64]: I will restrict seqspec to file_format yaml, but don't think other restrictions are necessary.

**Review by shengchd — APPROVED** [review]
Looks good to me.

**Review by shengchd — APPROVED** [review]

**Pull request merged**